pixel/theme.json

```diff
@@ -276,6 +276,41 @@
                         "left": "0.25rem !important",
                         "right": "0.25rem !important"
                     }
+                },
+                "elements": {
+                    "link": {
+                        ":active": {
+                            "typography": {
+                                "textDecoration": "underline dotted"
+                            }
+                        },
+                        ":focus": {
+                            "typography": {
+                                "textDecoration": "underline solid"
+                            }
+                        }
+                    }
+                }
+            },
+            "core/post-content": {
```

Comment:
Based on the Figma, links inside post content should be italicized in :active, :focus:, :hover states. I added a config for that here, but I noticed that it broke the block margins even though it should be a valid config: 

<img width="1624" alt="Screen Shot 2022-09-07 at 10 39 28 AM" src="https://user-images.githubusercontent.com/5375500/188906487-798388bf-34a8-4bac-9f11-bbd27bfa40c6.png">

Any idea why? I will investigate and open a bug report upstream if needed, but we may need to find a workaround in the meantime.

Reply:
Reported here: https://github.com/WordPress/gutenberg/issues/43955

Reply:
As a workaround added explicit `spacing -> blockGap` in `post-content` to fix the issue.